test/plugin/test_out_google_cloud.rb

```diff
@@ -199,6 +186,38 @@ def test_gce_metadata_does_not_load_when_fetch_gce_metadata_is_false
 
   def test_one_log
     setup_logging_stubs
+    d = create_driver()
+    d.emit({'message' => log_entry(0)})
+    d.run
+    verify_log_entries(1, COMPUTE_PARAMS)
+  end
+
+  def test_one_log_with_json_credentials
+    setup_logging_stubs
+    ENV['GOOGLE_APPLICATION_CREDENTIALS'] = 'test/plugin/data/credentials.json'
+    d = create_driver()
+    d.emit({'message' => log_entry(0)})
+    d.run
+    verify_log_entries(1, COMPUTE_PARAMS)
+  end
+
+  def test_one_log_with_invalid_json_credentials
+    setup_logging_stubs
+    ENV['GOOGLE_APPLICATION_CREDENTIALS'] = 'test/plugin/data/invalid_credentials.json'
+    d = create_driver()
+    d.emit({'message' => log_entry(0)})
+    exception_count = 0
+    begin
+      d.run
+    rescue RuntimeError => error
+      assert error.message.include? 'Unable to read the credential file'
+      exception_count += 1
```

Comment:
I think one or the other is sufficient, correct?  The one test that does 'assert false' is using it for brevity, but I think I'll change it for consistency.  All of those 'exception_count = 0' statements are useless, though, so I'll rearrange them too.